src/utils/handleTicketDelete.ts

```diff
@@ -95,21 +98,18 @@ export const handleTicketDelete = async (
 				if (!logsChannel?.isText()) return;
 				if (!logsChannel.permissionsFor(interaction.guild!.me!).has(['SEND_MESSAGES'])) return;
 
-				const { cache } = interaction.channel.messages;
-				const messages = cache.map((msg) => {
-					if (msg.author?.id !== interaction.client.user!.id) {
-						const user = msg.author?.tag ?? 'Unknown';
-						const content = msg.content ?? 'No message content';
-
-						return `${user}: ${content}\n`;
+				const allMessages = await fetchMessages(interaction.channel, []);
```

Comment:
```suggestion
				const allMessages = await fetchMessages(interaction.channel);
```
View suggestion at line 157 for info